src/test/java/org/sqlite/DBMetaDataTest.java

```diff
@@ -1001,7 +999,7 @@ public void columnOrderOfgetTables() throws SQLException {
         ResultSet rsTables =
                 meta.getTables(
                         null,
-                        null,
+                        "",
```

Comment:
I'm not sure about that, i don't think passing `""` should match any schema, that's what passing `null` is for.